frontend/lib/Providers/group_list_provider.dart

```diff
@@ -32,4 +32,8 @@ class GroupListProvider with ChangeNotifier {
     groupsList.removeWhere((element) => element.id == id);
     notifyListeners();
   }
+
+  void notify(){
+    notifyListeners();
+  }
```

Comment:
`notifyListeners` nie powinno być w taki sposób udostępniane innym klasom, to `Provider` jest odpowiedzialny za powiadomienie innych, że jego stan się zmienił

Reply:
Nervermind, to nie będzie możliwe ze względu na mechanizm sortowania